backend/main.py

```diff
@@ -87,6 +100,12 @@ class TestForm(FormPage):
             divider: Divider
             label: Label = "Label"
             hidden: Hidden = "Hidden"
+            dropdown: DropdownChoices = (
+                "2"  # When there are > 3 choices a dropdown will be rendered
+            )
+            radio: RadioChoices = (
+                "3"  # When there are <= 3 choices a radio group will be rendered
+            )
```

Comment:
move comment to above the `prop`, now it seemed like you were talking about the `"2"` and `"3"` value.
```suggestion
            # When there are > 3 choices a dropdown will be rendered
            dropdown: DropdownChoices = ("2")
            # When there are <= 3 choices a radio group will be rendered
            radio: RadioChoices = ("3")
```